phoenix-core/src/main/protobuf/PTable.proto

```diff
@@ -106,19 +106,16 @@ message PTable {
   optional int32 viewIndexIdType = 39 [default = 5];
   optional bool viewModifiedUpdateCacheFrequency = 40;
   optional bool viewModifiedUseStatsForParallelization = 41;
-  optional int64 phoenixTTL = 42;
```

Comment:
Let's not change the message format in this release. Maybe safer to remove it in a later release when the columns are removed. We will need to ensure backward and forward compatibility of messages against various versions of the client and server.